zookeeper/src/test/java/com/linecorp/armeria/client/zookeeper/TestBase.java

```diff
@@ -49,7 +49,9 @@
     @BeforeClass
     public static void start() {
         try {
-            zkInstance = ZKFactory.apply().withRootDir(ROOT_FOLDER.newFolder("zookeeper")).create();
+            if (zkInstance == null) {
```

Comment:
Can you explain what's going on here (maybe also adding a comment)?

I would expect the stop() in AfterClass to make this logic unnecessary so it's not clear.